PayloadPkg/FirmwareUpdate/GetCapsuleImage.c

```diff
@@ -269,12 +300,25 @@ LoadCapsuleImage (
   FileHandle        = NULL;
   HwPartHandle      = NULL;
   FsHandle          = NULL;
+  SwPart		    = 0;
+  BootSlot = 0;
```

Comment:
Align with previous variable initializations, and use spaces instead of tab characters.